compat/Makefile.am

```diff
@@ -1,7 +1,5 @@
 dist_bin_SCRIPTS = ubuntu-core-launcher
 CLEANFILES = $(dist_bin_SCRIPTS)
-EXTRA_DIST = ubuntu-core-launcher.in
 
-ubuntu-core-launcher: ubuntu-core-launcher.in Makefile
-	sed -e 's,[@]libexecdir[@],$(libexecdir),g' < $(srcdir)/ubuntu-core-launcher.in > ubuntu-core-launcher
-	chmod +x ubuntu-core-launcher
+ubuntu-core-launcher: Makefile
+	ln -s snap-confine ubuntu-core-launcher
```

Comment:
Note that the location of snap-confine and ubuntu-core-launcher can be rather different. I will need to experiment with how this affects downstream distributions. Some packages (*ahem* dpkg) don't deal very well with file/symlink transitions AFAIR